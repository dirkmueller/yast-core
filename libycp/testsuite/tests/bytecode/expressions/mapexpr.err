{
    // filename: "tests/bytecode/expressions/mapexpr.ycp"
    return $[(1 + 1):("a" + "a"), (2 + 2):("b" + "b"), (3 + 3):("c" + "c")];
}
