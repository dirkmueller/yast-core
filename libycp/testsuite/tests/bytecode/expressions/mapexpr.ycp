{
    return $[(1 + 1):("a" + "a"), (2 + 2):("b" + "b"), (3 + 3):("c" + "c")];
}
