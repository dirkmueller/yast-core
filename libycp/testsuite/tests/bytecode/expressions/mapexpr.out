$[2:"aa", 4:"bb", 6:"cc"]
